docs/dev_ops/tests.md

```diff
@@ -0,0 +1,25 @@
+# Testing Modalities
+
+Modalities has a threefold setup for testing, namely
+
+* Main tests <br> 
+The main tests comprise CPU, single GPU and multi-GPU tests. The latter ones create a distributed environment internally and allow end2end testing of modalities. 
+Each of these tests defines its requirements (typically the number of GPUs) in the test and the test will be skipped if the requirements are not met.
+
+* Torchrun tests <br>
+These tests are run from a shell script using torchrun and are typically end2end or at least integration tests. Since we implemented distributed testing using multiprocessing within modalities, these tests will be integrated into the main tests in the long term. Note that some of the torchrun tests have been already migrated to the main tests. 
```

Comment:
```suggestion
These tests are run from a shell script using torchrun and are typically end2end or at least integration tests. Since we implemented distributed testing using multiprocessing within Modalities, these tests will be integrated into the main tests in the long term. Note that some of the torchrun tests have been already migrated to the main tests. 
```